client/src/document/ingredients/browser-compatibility-table/legend.tsx

```diff
@@ -73,7 +74,14 @@ function getActiveLegendItems(
         if (versionSupport.prefix) {
           legendItems.add("prefix");
         }
-        if (versionSupport.notes) {
+        if (versionSupport.notes && Array.isArray(browserSupport)) {
+          legendItems.add("chevron");
+        }
+        if (
+          versionSupport.notes &&
+          !versionSupport.version_removed &&
+          !versionSupport.partial_implementation
+        ) {
```

Comment:
Oh, so basically we only show the notes icon if we have notes in combination with "Full support", because otherwise (Partial/no support), we usually find a note anyways. Is that right?

Reply:
Yes, more or less. All partial support items have notes. See https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/API/browserAction/getTitle#browser_compatibility. Everything there has full support with no notes, partial support with notes, or no support, but currently the symbol appears anyway.

The check for version removed is because this symbol should only appear in the table for active notes (getFirst(support) is done for it, after all).